### DISCUSSION TITLE
Start modded natively through steam?

### DISCUSSION THREAD

**Discussion by saminlatif50** [ebkr/r2modmanPlus#612] [Q&A] [answered]
I'm using r2modman to run mods for H3VR and so far its been running very smoothly, but I was wondering if there was any option on launching the game modded through the Steam interface similar to how the old DeliCounter for Bonetome worked? I couldn't find any documentation of it online or in the app. Thanks!

**Comment by ebkr** [comment, marked as answer]
Check the help section in the manager.
There’s a bit on launching through Steam.

Note that it’s not recommended because you’ll lose out on profile functionality unless you manually update the launch parameters each time, and you also won’t be aware of any mod updates (because let’s be honest, you’ll only check occasionally when you want a new mod 👀).

**Reply by saminlatif50** [reply]
Sounds good, thank you!
